yarn-project/boxes/private-token/src/scripts/util.ts

```diff
@@ -0,0 +1,41 @@
+import { AccountWallet, getSandboxAccountsWallets } from '@aztec/aztec.js';
+import { FunctionAbi, encodeArguments } from '@aztec/foundation/abi';
+import { AztecRPC, CompleteAddress } from '@aztec/types';
+// hack: addresses are stored as string in the form to avoid bigint compatibility issues with formik
+// convert those back to bigints before sending
+export function convertArgs(functionAbi: FunctionAbi, args: any, encode: boolean = true) {
```

Comment:
We need this because we are using formik. This function should live side by side with the function getting values from formik. And the scripts should only take args that are correct types. So that if we replace formlik with something else, we don't have to change the scripts.